packages/gatsby-plugin-image/src/node-apis/image-processing.ts

```diff
@@ -85,10 +85,11 @@ export async function writeImages({
     async ([hash, { src, ...args }]) => {
       let file: FileSystemNode | undefined
       let fullPath
-      if (process.env.GATSBY_EXPERIMENTAL_REMOTE_IMAGES && isRemoteURL(src)) {
+      if (isRemoteURL(src)) {
         let createRemoteFileNode
         try {
-          ;({ createRemoteFileNode } = require(`gatsby-source-filesystem`))
```

Comment:
Yeah, I know, @laurieontech. This is just easier to read.

Reply:
Haha I agree, don’t worry

Reply:
I mean, I'm grateful that I now know how to do it!